src/main/java/duke/commands/TaskCommand.java

```diff
@@ -0,0 +1,50 @@
+package duke.commands;
+import java.util.Arrays;
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+import duke.tasks.Task;
+import duke.tasks.ToDo;
+import duke.tasks.Deadline;
+import duke.tasks.Event;
+import duke.status.typeTask;
+import duke.exceptions.WrongDateFormatException;
+import duke.exceptions.WrongEventOrDeadlineFormatException;
+import duke.exceptions.WrongTimeFormatException;
+
+public class TaskCommand extends Command {
+
+    public TaskCommand(String desc) {
+        super(desc);
+    }
+
+    @Override
+    public boolean isExit() {
+        return false;
+    }
+
+    @Override
+    public void execute(
+            TaskList taskList, Ui ui,
+            Storage storage) throws WrongDateFormatException,
+            WrongTimeFormatException,
+            WrongEventOrDeadlineFormatException {
+        String[] instructions = this.commandDescription.split(" ");
+        String importantInstructions = String.join(" ",
+                Arrays.copyOfRange(instructions,
+                        1, instructions.length));
+        String identity = instructions[0];
+        Task task;
+        if (identity.equals(typeTask.TODO.getTask())) {
+            task = new ToDo(importantInstructions);
+        } else if (identity.equals(typeTask.DEADLINE.getTask())) {
+            task = new Deadline(importantInstructions);
+        } else {
+            task = new Event(importantInstructions); 
+        }
```

Comment:
Don't use the else statement when you can use an if statement. See [this](https://nus-cs2103-ay2122s1.github.io/website/se-book-adapted/chapters/codeQuality.html#use-the-default-branch)